Index: configure.ac
===================================================================
RCS file: /cvsroot/gaim/gaim/configure.ac,v
retrieving revision 1.360
diff -u -b -B -r1.360 configure.ac
--- configure.ac        17 Dec 2005 21:28:54 -0000      1.360
+++ configure.ac        18 Dec 2005 15:22:33 -0000
@@ -128,11 +128,18 @@
 HOWL_CFLAGS=""
 HOWL_LIBS=""

-dnl Attempt to autodetect Howl
-PKG_CHECK_MODULES(HOWL, howl,
+dnl First we attempt to use avahi-compat-howl
+PKG_CHECK_MODULES(HOWL, avahi-compat-howl,
        [howlincludes="yes" howllibs="yes"],
        [howlincludes="no" howllibs="no"])

+dnl If that fails, autodetect the "real" howl
+if test "$howlincludes" = "no" ; then
+        PKG_CHECK_MODULES(HOWL, howl,
+                [howlincludes="yes" howllibs="yes"],
+                [howlincludes="no" howllibs="no"])
+fi
+
 dnl Override HOWL_CFLAGS if the user specified an include dir
 if test "$ac_howl_includes" != "no"; then
        HOWL_CFLAGS="-I$ac_howl_includes"
